scripts/seeding/seeding_prep.py

```diff
@@ -0,0 +1,239 @@
+from math import sqrt
+
+from great_ape_safe import GreatApeSafe
+from great_ape_safe.ape_api.helpers.uni_v3.uni_v3_sdk import Q96
+
+from brownie import accounts, interface
+from helpers.addresses import r
+from rich.console import Console
+
+C = Console()
+
+"""
+Funds allocation (univ3):
+* 0.925-0.99 eBTC/WBTC: 10% of funds allocated to liquidity provision
+* 0.99-1 eBTC/WBTC: 40% of funds allocated to liquidity provision
+* 1-1.01 eBTC/WBTC: 40% of funds allocated to liquidity provision
+* 1.01-1.08 eBTC/WBTC: 10% of funds allocated to liquidity provision
+"""
+
+# misc.univ3
+FEE_TIER = 500
+PCT_10 = 0.1
+PCT_40 = 0.4
+
+PRICE_0925 = 0.925
+PRICE_099 = 0.99
+PRICE_1 = 1
+PRICE_101 = 1.01
+PRICE_108 = 1.08
+
+# misc.cow
+COEF = 0.98
+DEADLINE = 60 * 60 * 3
+
+# misc.ebtc
+COLLATERAL_TARGET_RATIO = 200
+
+safe = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+safe.init_uni_v3()
+
+
+def prep():
+    safe.init_cow(prod=True)
+
+    # tokens
+    wbtc = safe.contract(r.assets.wbtc)
+    ebtc = safe.contract(r.assets.ebtc)
+    reth = safe.contract(r.assets.reth)
+    steth = safe.contract(r.assets.steth)
+
+    # =========== $reth -> $steth swap (happens async) ===========
+    safe.cow.market_sell(
+        reth, steth, reth.balanceOf(safe), deadline=DEADLINE, coef=COEF
```

Comment:
As per TCD, we should reduce this amount to 800. Further expansion will be discussed separately.

Reply:
solved in 075c04028a69f00cffb48521877d45cbf7fb2ab7